beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -1119,6 +1143,8 @@ impl<T: BeaconChainTypes> BeaconProcessor<T> {
                             self.spawn_worker(item, toolbox);
                         } else if let Some(item) = bbrange_queue.pop() {
                             self.spawn_worker(item, toolbox);
+                        } else if let Some(item) = lcbootstrap_queue.pop() {
+                            self.spawn_worker(item, toolbox);
```

Comment:
This `if` chain is a priority list, so I think we should move light client stuff right to the bottom, after `backfill_chain_segment`.